Com.OneSignal.iOS/OneSignalImplementation.cs

```diff
@@ -145,6 +145,11 @@ public override void DeleteTags(IList<string> keys)
       iOS.OneSignal.DeleteTags(objs);
     }
 
+    public override void ClearOneSignalNotifications()
+    {
+       throw new NotImplementedException();
```

Comment:
Would be better to log or silently do nothing. Otherwise the app developer will need to add an if platform == Android around this call

Reply:
A log would be best. Or even better call
```
            UIKit.UIApplication.SharedApplication.ApplicationIconBadgeNumber = 0;
```
Which clears the badge on iOS